test/jdk/java/nio/channels/DatagramChannel/SRTest.java

```diff
@@ -24,184 +24,216 @@
 /* @test
  * @summary Test DatagramChannel's send and receive methods
  * @author Mike McCloskey
```

Comment:
This change is essentially replacing this JDK 1.4 era test so probably should remove the @author tag.

Reply:
Removed author tag, marking as resolved +1